pkg/eventfilter/subscriptionsapi/exact_filter_test.go

```diff
@@ -139,3 +140,28 @@ func makeEventWithExtension(extName, extValue string) *cloudevents.Event {
 	e.SetExtension(extName, extValue)
 	return e
 }
+
+func TestExactTimeMatchIsStable(t *testing.T) {
+	event := cetest.FullEvent()
+	time, err := event.Time().MarshalText()
+	if err != nil {
+		t.Fatalf("error while marshalling time to text: %v", err)
+	}
+	f, err := NewExactFilter(map[string]string{
+		"time": string(time),
+	})
+	if err != nil {
+		t.Fatalf("failed to create filter")
+	}
+	// the filter should consistently pass here, let's check that it does
+	failCount := 0
+	for i := 0; i < 10_000; i++ {
```

Comment:
How long it takes for 10k iterations? perhaps a smaller number is enough?

Reply:
seems like a reasonable number to start with?